brainstormforce/header-footer-elementor
# Fix: Navigation Menu - Fixed active color issues in case of full page anchor links used in nav links.

### Description
Navigation menu - Fixed active color issue in case of full page anchor links are used in nav links.

### Screenshots


### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
1. Use custom links like `https://example.com/page#anchor`
2. Apply colors from available controls (Normal/Hover/Active)
3. Check on the frontend if all links get active color applied or the only current page link.

### Checklist:
